project/web/gunicorn.conf

```diff
@@ -7,9 +7,8 @@ autorestart=true
 stopasgroup=true
 killasgroup=true
 stopwaitsecs=60
-stdout_logfile={{ log_dir }}/gunicorn.log
-redirect_stderr=true
-stderr_logfile={{ log_dir }}/gunicorn.error.log
+stdout_syslog=true
```

Comment:
Is `stdout_syslog` valid for the version of supervisor we're using? (I think we're using 3.0). The online docs don't mention it, but I do see it in github, but for version 4.0:
https://github.com/Supervisor/supervisor/blob/master/docs/configuration.rst

![image](https://cloud.githubusercontent.com/assets/4413/9816409/d569ab2a-586a-11e5-9d6c-d59106a15135.png)